skrf/vectorFitting.py

```diff
@@ -689,46 +719,57 @@ def plot_pz(self, i, j):
         j : int
             Column index of the response.
 
+        ax : :class:`matplotlib.axes.AxesSubplot` object or None
+            matplotlib axes to draw on. If None, the current axes is fetched with gca().
+
         Returns
         -------
         None
         """
 
+        if ax is None and mplt is not None:
```

Comment:
The mplt check is unnecessary here, it's already checked by the @check_plotting decorator

Reply:
You're right, thanks! I updated those checks in https://github.com/scikit-rf/scikit-rf/pull/450/commits/4b8d25947e65e8cf6167886f1fb91300f5ae1c1a